src/RadixDlt.NetworkGateway.DataAggregator/Workers/NodeWorkers/NodeTransactionFetchWorker.cs

```diff
@@ -175,6 +175,15 @@ private async Task FetchTransactions(CancellationToken cancellationToken)
             fetchMaxBatchSize
         );
 
+        if (batchSize == 0)
```

Comment:
This fixes a misc bug I noted where the Aggregator was requesting 0 transactions from the Core API